modules/qna/src/backend/api.ts

```diff
@@ -47,7 +47,7 @@ export default async (bp: typeof sdk, bots: ScopedBots) => {
       const question = await storage.getQnaItem(req.params.id)
       res.send(question)
     } catch (e) {
-      sendToastError('Fetch', e.message)
+      res.status(404).send(e.message || 'Error')
```

Comment:
When a QnA was not found, the `sendToasError` sent a message on the websocket and left the HTTP request open, making it impossible for the React client to handle the error properly. Responding with a 404 makes more sense.

Reply:
agreed